src/pages/signIn/email/index.tsx

```diff
@@ -59,9 +59,13 @@ export const Email = () => {
           </div>
           <div className="text-center">
             <span className={css['form__subtitle']}>{translate('accept')}</span>
-            <Link href="/" label={translate('terms-of-use')} target="_blank" />
+            <Link
+              href="https://webapp2.dev.socious.io/privacy-policy/"
```

Comment:
Why dev ? 
Shouldn't create a `env` variable for production and dev of socious app?

Reply:
dev is fixed,
But since this URL is used just here for linking to another source, I think this is not needed to be set in env variables. Because then we should define these variables in both dev and prod server, and it is likely that we have shin dedicated 'privacy policy' page soon